docs/cmdline-opts/gen.pl

```diff
@@ -48,8 +48,13 @@
 my %catlong;
 
 use POSIX qw(strftime);
-my $date = strftime "%B %d %Y", localtime;
-my $year = strftime "%Y", localtime;
+if (defined($ENV{SOURCE_DATE_EPOCH})) {
+    my $date = strftime "%B %d %Y", localtime($ENV{SOURCE_DATE_EPOCH});
+    my $year = strftime "%Y", localtime($ENV{SOURCE_DATE_EPOCH});
+} else {
+    my $date = strftime "%B %d %Y", localtime;
+    my $year = strftime "%Y", localtime;
+}
```

Comment:
```suggestion
if (defined($ENV{SOURCE_DATE_EPOCH})) {
    my $ts = localtime($ENV{SOURCE_DATE_EPOCH};
} else {
    my $ts = localtime;
}
my $date = strftime "%B %d %Y", ts;
my $year = strftime "%Y", ts;
```
This can remove some repetition.

Reply:
Applied, thanks.